tests/test_regex_identifier.py

```diff
@@ -1,10 +1,46 @@
-from datetime import datetime
-from time import time
+import re
 
 import pytest
 
 from pywhat import regex_identifier
 from pywhat.filter import Filter
+from pywhat.helper import load_regexes
+
+
+def test_if_all_tests_exist():
+    database = load_regexes()
+
+    with open("tests/test_regex_identifier.py", "r", encoding="utf-8") as file:
+        tests = file.read()
+
+    for regex in database:
+        assert (
+            regex["Name"] in tests
+        ), "No test for this regex found in 'test_regex_identifier.py'. Note that a test needs to assert the whole name."
+
+
+def test_regex_format():
+    database = load_regexes()
+
+    for regex in database:
+        assert re.findall(
+            r"^(?:\(\?i\))?\^\(.*\)\$$", regex["Regex"]
+        ), r"Please use ^(regex)$ regex format. If there is '\n' character, you have to escape it. If there is '(?i)', it is allowed and should be before the '^'."
+
+        assert (
+            re.findall(r"\^\||\|\^|\$\|\^|\$\||\|\$", regex["Regex"]) == []
+        ), "Remove in-between boundaries. For example, '^|$' should only be '|'."
+
+
+def test_sorted_by_rarity():
+    database = load_regexes()
+    rarity_num = []
```

Comment:
You can simplify this by doing `rarity_num = [regex["Rarity"] for regex in database]`